peer.py

```
@@ -77,58 +105,32 @@ async def main():
 
-            # 응답 보내기
-            response = f"ACK FROM {peer_id}"
+            # 메시지 처리 - PING 요청이면 PONG으로 응답
+            if message.startswith("PING"):
+                try:
+                    # PING 메시지에서 피어 ID 추출
+                    sender_id = message.split()[1]
+                    # 피어 상태 업데이트
+                    peer_status_cache[sender_id] = time.time()
+                    response = f"PONG {peer_id}"
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)